besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2352,16 +2355,46 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
   }
 
   private GenesisConfigFile getGenesisConfigFile() {
-    return GenesisConfigFile.fromConfig(genesisConfig());
+    return GenesisConfigFile.fromConfigWithoutAccounts(genesisConfig());
   }
 
+  private String genesisConfigString = "";
```

Comment:
[A `Suppliers.memoize` ](https://guava.dev/releases/snapshot-jre/api/docs/com/google/common/base/Suppliers.html#memoize(com.google.common.base.Supplier)) seems better suited for this field

Reply:
I made a commit to use Suppliers.memoize: https://github.com/hyperledger/besu/pull/6952/commits/8ed9e4c32f203f8bb75479015d3d92e49e6b98b6